src/pseudopeople/schema_entities.py

```diff
@@ -1,31 +1,221 @@
+        "middle_initial",
+        (
+            NOISE_TYPES.MISSING_DATA,
+            NOISE_TYPES.PHONETIC,
+            NOISE_TYPES.OCR,
+            NOISE_TYPES.TYPOGRAPHIC,
+        ),
+    )
+    RACE_ETHNICITY: Column = Column(
+        "race_ethnicity",
+        (
+            NOISE_TYPES.MISSING_DATA,
+            NOISE_TYPES.INCORRECT_SELECTION,
+        ),
+    )
+    RELATION_TO_HOUSEHOLD_HEAD: Column = Column(
+        "relation_to_household_head",
+        (
+            NOISE_TYPES.MISSING_DATA,
+            NOISE_TYPES.INCORRECT_SELECTION,
+        ),
+    )
+    SEX: Column = Column(
+        "sex",
+        (
+            NOISE_TYPES.MISSING_DATA,
+            NOISE_TYPES.INCORRECT_SELECTION,
+        ),
+    )
+    SSA_EVENT_DATE: str = "event_date"
+    SSA_EVENT_TYPE: str = "event_type"
+    SSN: str = "ssn"
+    STATE: Column = Column(
+        "state",
+        (
+            NOISE_TYPES.MISSING_DATA,
+            NOISE_TYPES.INCORRECT_SELECTION,
+        ),
+    )
+    STREET_NAME: Column = Column(
+        "street_name",
+        (
+            NOISE_TYPES.MISSING_DATA,
+            NOISE_TYPES.PHONETIC,
+            NOISE_TYPES.OCR,
+            NOISE_TYPES.TYPOGRAPHIC,
+        ),
+    )
+    STREET_NUMBER: Column = Column(
+        "street_number",
+        (
+            NOISE_TYPES.MISSING_DATA,
+            NOISE_TYPES.NUMERIC_MISWRITING,
+            NOISE_TYPES.OCR,
+            NOISE_TYPES.TYPOGRAPHIC,
+        ),
+    )
+    TAX_FORM: str = "tax_form"
+    UNIT_NUMBER: Column = Column(
+        "unit_number",
+        (
+            NOISE_TYPES.MISSING_DATA,
+            NOISE_TYPES.NUMERIC_MISWRITING,
+            NOISE_TYPES.OCR,
+            NOISE_TYPES.TYPOGRAPHIC,
+        ),
+    )
+    ZIPCODE: Column = Column(
+        "zipcode",
+        (
+            NOISE_TYPES.MISSING_DATA,
+            NOISE_TYPES.ZIPCODE_MISWRITING,
+            NOISE_TYPES.OCR,
+            NOISE_TYPES.TYPOGRAPHIC,
+        ),
+    )
 
-COLUMNS = __Columns()
+
+@dataclass
+class Form:
+    name: str
+    columns: Tuple[Column] = None
+    row_noise_types: Tuple[RowNoiseType] = (NOISE_TYPES.OMISSION, NOISE_TYPES.DUPLICATION)
+    is_implemented: bool = True
+
+
+class __Forms(NamedTuple):
+    """NamedTuple that contains information about forms and their related columns"""
+
+    CENSUS: Form = Form(
+        "decennial_census",
+        columns=(
+            NoisedColumns.FIRST_NAME,
+            NoisedColumns.MIDDLE_INITIAL,
+            NoisedColumns.LAST_NAME,
+            NoisedColumns.AGE,
+            NoisedColumns.DOB,
+            NoisedColumns.STREET_NUMBER,
+            NoisedColumns.STREET_NAME,
+            NoisedColumns.UNIT_NUMBER,
+            NoisedColumns.CITY,
+            NoisedColumns.STATE,
+            NoisedColumns.ZIPCODE,
+            NoisedColumns.RELATION_TO_HOUSEHOLD_HEAD,
+            NoisedColumns.SEX,
+            NoisedColumns.RACE_ETHNICITY,
+        ),
+    )
+    ACS: Form = Form(
```

Comment:
All forms but 1040 are implemented now, right? Why are they set to `is_implemented=False`?

Reply:
Oh I forgot to mention in slack (and you weren't in standup) - we decided to open this ticket with the caveat that I will definitely not merge until all forms are implemented